packages/infra/lib/api-stack/document-upload.ts

```diff
@@ -0,0 +1,51 @@
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import * as ecs_patterns from "aws-cdk-lib/aws-ecs-patterns";
+import { ILayerVersion } from "aws-cdk-lib/aws-lambda";
+import { S3EventSource } from "aws-cdk-lib/aws-lambda-event-sources";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { Construct } from "constructs";
+import { EnvType } from "../env-type";
+import { createLambda as defaultCreateLambda } from "../shared/lambda";
+
+export function createLambda({
+  lambdaLayers,
+  stack,
+  vpc,
+  apiService,
+  envType,
+  medicalDocumentsUploadBucket,
+  medicalDocumentsBucket,
+  sentryDsn,
+}: {
+  lambdaLayers: ILayerVersion[];
+  stack: Construct;
+  vpc: ec2.IVpc;
+  apiService: ecs_patterns.NetworkLoadBalancedFargateService;
+  envType: EnvType;
+  medicalDocumentsBucket: s3.IBucket;
+  medicalDocumentsUploadBucket: s3.Bucket;
+  sentryDsn: string | undefined;
+}) {
+  const documentUploaderLambda = defaultCreateLambda({
+    stack,
+    name: "DocumentUploader",
+    vpc,
+    entry: "document-uploader",
+    layers: lambdaLayers,
+    envVars: {
+      ENV_TYPE: envType,
+      API_URL: `http://${apiService.loadBalancer.loadBalancerDnsName}/internal/docs/doc-ref`,
```

Comment:
The more recent lambdas get only the base url, leaving the actual endpoint being called to the lambda to have on code.

It helps decoupling app logic from the underlying infra.